include/CMakeLists.txt

```diff
@@ -11,7 +13,8 @@ add_library(
 )
 
 target_include_directories(
-    atomic_queue
+    ${PROJECT_NAME}
```

Comment:
An explicit target name should be used here (as it was being used before), we should not assume that `${PROJECT_NAME}` is the same as the target name passed to `add_library`.

Reply:
👍